src/NHSDPublicBrowseAcceptanceTestsSpecflow/Steps/BrowseSolutions/ViewASolution.cs

```diff
@@ -112,5 +120,29 @@ public void ThenThePageWillContainAnIndicationThatTheSolutionMeetsTheCriteriaFor
             _test.pages.ViewASolution.FoundationSolutionIndicatorDisplayed().Should().BeTrue();
         }
 
+        [Then(@"the capabilities listed match the expected capabilities in the database")]
+        public void ThenTheCapabilitiesListedMatchTheExpectedCapabilitiesInTheDatabase()
+        {
+            var solutionId = _test.pages.ViewASolution.GetSolutionId();
+            var capabilities = SqlHelper.GetSolutionCapabilities(solutionId, _test.connectionString).Split(',').ToList();
+            var actualCapabilities = _test.pages.ViewASolution.GetSolutionCapabilities();
+            actualCapabilities.Should().BeEquivalentTo(capabilities);
+        }
+
+        [Then(@"the Download more information button downloads a '(.*)' file")]
+        public void ThenTheDownloadMoreInformationButtonDownloadsAFile(string fileFormat)
+        {
+            // Filename is to match the solution ID at all times
+            var solId = _test.pages.ViewASolution.GetSolutionId();
+            var fileName = $"{solId}.{fileFormat.ToLower()}";
+            var downloadLink = _test.pages.ViewASolution.GetDownloadUrl();
+
+            downloadLink.Should().Contain(fileName);
+            
+            // Does the download. Will fail test if download fails
+             using(WebClient client = new WebClient()){
+                client.DownloadFile(downloadLink, $"C:\\temp\\{fileName}");
```

Comment:
Can this be placed in the current working directory?
Running on Linux build agents won't like the C:\\ drive

Reply:
Fixed in commit 3e587fe